### PR TITLE
coding-standard: drop Nette\Object ancesstor for all

### PR DESCRIPTION
**Pull request by TomasVotruba**
Probably not the case anymore, since Smart traits came and "Object" is a keyword

### CODE REVIEW

**Comment by dg** [comment]
What about change it to SmartObject or StaticClass?

**Comment by TomasVotruba** [comment]
> What about change it to SmartObject or StaticClass?

Can be, if that's the case in most of Nette objects.

Should I change it?

**Comment by dg** [comment]
Yes please :)

**Comment by TomasVotruba** [comment]
Done :+1: 

Is it ok?

File: www/en/coding-standard.texy [hunk 1]
```diff
@@ -206,7 +206,7 @@ Class members implementation should follow this order as it is practical:
   b) protected
   c) private
 
-The ultimate ancestor of all instantiable classes should be `Nette\Object`.
+All instantiable classes should use `Nette\SmartObject` or `Nette\StaticClass` traits.
```
Comment (dg) [inline www/en/coding-standard.texy]: All instantiable classes should use `Nette\SmartObject` trait and non-instantiable `Nette\StaticClass` trait.

**Comment by TomasVotruba** [comment]
Done

**Comment by dg** [comment]
Great, thanks

**Pull request merged**